ajax/libs/webpack-cesium/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webpack-cesium",
+  "version": "1.31.2",
+  "description": "webpack version of Cesium",
+  "filename": "webpack.cesium.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DigitalArsenal/webpack-cesium.git"
+  },
+  "keywords": [
+    "webpack",
+    "cesium",
+    "geospatial",
+    "globe",
+    "google",
+    "earth",
+    "wms",
+    "wmts",
+    "kml",
+    "czml",
+    "space",
+    "satellite",
+    "agi",
+    "stk"
+  ],
+  "author": "digitalarsenal.io",
```

Comment:
If this library was authored by a team, you can change this to `DigitalArsenal Team (http://digitalarsenal.io)`.
If it was authored a single person, you can mention his/her name here.
If you'd like to specify multiple authors you can use `"authors"` here.